eslint.yaml

```diff
@@ -35,7 +35,7 @@ rules:
   vars-on-top: 2
   no-debugger: 2
   no-cond-assign: 2
-  no-console: 2
+  no-console: 1
```

Comment:
I would rather see instances were console must be used be eslint ignored than change this global setting